editoast/src/models/pathfinding.rs

```diff
@@ -18,10 +18,21 @@ use serde::{Deserialize, Serialize};
 ///
 /// It differs from infra/pathfinding that performs a topological pathfinding
 /// meant to be used with the infra editor.
-#[derive(Debug, Clone, PartialEq, Serialize, Derivative, Queryable, QueryableByName, Model)]
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Serialize,
+    Deserialize,
+    Derivative,
+    Queryable,
+    QueryableByName,
+    Model,
+    Insertable,
+)]
 #[derivative(Default(new = "true"))]
 #[model(table = "osrd_infra_pathmodel")]
-#[model(retrieve, delete)]
+#[model(create, retrieve, delete)]
```

Comment:
Where is the create used? It's probably not ideal since you would have to manually specify the ID instead of letting Postgres choose. I'd suggest removing it and using `PathfindingChangeset::create()` and then `.into()` it into a regular `Pathfinding`.

Reply:
Probably not needed !